### PR TITLE
DOCS-10106 fix prerequisites

### PR DESCRIPTION
**Pull request by ferferrarazzo**

### CODE REVIEW

**Review by ferferrarazzo — COMMENTED** [review]
OK

File: modules/ROOT/pages/installation-prerequisites.adoc [hunk 1]
```diff
@@ -6,34 +6,71 @@ endif::[]
 
 To install Anypoint API Community Manager you will need:
 
-. An Anypoint admin user account.
-. A Salesforce API Community Manager organization admin user with a role.
-+
+== A Salesforce Admin User
```
Comment (kbostrom) [inline modules/ROOT/pages/installation-prerequisites.adoc]: The following is a task and so it requires a task title, something like...:
```suggestion
== Update Your Access to Become a Salesforce Admin User
```

File: modules/ROOT/pages/installation-prerequisites.adoc [hunk 2]
```diff
@@ -6,34 +6,71 @@ endif::[]
...
+
+Installation requires a Salesforce API Community Manager organization admin user with a role.
```
Comment (kbostrom) [inline modules/ROOT/pages/installation-prerequisites.adoc]: passive. Try...:
```suggestion
To install Anypoint API Community Manager, you must update your roles to become a Salesforce API Community Manager organization Admin user:
```

File: modules/ROOT/pages/installation-prerequisites.adoc [hunk 3]
```diff
@@ -6,34 +6,71 @@ endif::[]
...
+
 To update your user role and permissions:
```
Comment (kbostrom) [inline modules/ROOT/pages/installation-prerequisites.adoc]: omit
```suggestion
```

File: modules/ROOT/pages/installation-prerequisites.adoc [hunk 4]
```diff
@@ -6,34 +6,71 @@ endif::[]
...
 
-.. Log in to your Salesforce organization.
-.. In *Setup*, search for *Users* using the *Quick Find* box and click *Users*.
-.. Locate your username and click *Edit*.
-.. Set the *Role* for your account.
+. Log in to your Salesforce organization.
+. In *Setup*, search for *Users* using the *Quick Find* box and click *Users*.
```
Comment (kbostrom) [inline modules/ROOT/pages/installation-prerequisites.adoc]:
```suggestion
. In *Setup*, use the *Quick Find* box to find and click *Users*.
```

File: modules/ROOT/pages/installation-prerequisites.adoc [hunk 5]
```diff
@@ -6,34 +6,71 @@ endif::[]
...
+. Locate your username and click *Edit*.
+. Set the *Role* for your account.
```
Comment (kbostrom) [inline modules/ROOT/pages/installation-prerequisites.adoc]: How does the user "set" the role? Perhaps specify something like "Set the role for your account by selecting from the *Role* list"?

File: modules/ROOT/pages/installation-prerequisites.adoc [hunk 6]
```diff
@@ -6,34 +6,71 @@ endif::[]
...
 +
 This can be any role from the list, but cannot be none.
 +
 If you don't see any roles in the list, refer to the Salesforce knowledge article https://help.salesforce.com/articleView?language=en_US&type=1&mode=1&id=000322715[Add roles to the role hierarchy, role=external, window=_blank] for instructions to create a role in your organization.
+. Select *Salesforce CRM Content User*.
+. *Save*.
```
Comment (kbostrom) [inline modules/ROOT/pages/installation-prerequisites.adoc]:
```suggestion
. Click *Save*.
```

File: modules/ROOT/pages/installation-prerequisites.adoc [hunk 7]
```diff
@@ -6,34 +6,71 @@ endif::[]
...
+
+== API Community Manager Entitlement
+
+Installation requires Anypoint API Community Manager entitlement for your Anypoint organization.
```
Comment (kbostrom) [inline modules/ROOT/pages/installation-prerequisites.adoc]: I suggest that you use active voice here and mimic the intro to the previous topic.

File: modules/ROOT/pages/installation-prerequisites.adoc [hunk 8]
```diff
@@ -6,34 +6,71 @@ endif::[]
...
 To check the entitlement:
 
-.. Log in to your Anypoint Platform organization.
-.. If you are using the US cloud, navigate to `+https://anypoint.mulesoft.com/accounts/api/profile+`.
+. Log in to your Anypoint Platform organization.
+. If you are using the US cloud, navigate to `+https://anypoint.mulesoft.com/accounts/api/profile+`.
 +
 If you are using the EU cloud, navigate to `+https://eu1.anypoint.mulesoft.com/accounts/api/profile+`.
 +
 If you are using MuleSoft Government Cloud, navigate to `+https://gov.anypoint.mulesoft.com/accounts/api/profile+`.
```
Comment (kbostrom) [inline modules/ROOT/pages/installation-prerequisites.adoc]: This is not the proper format for a step with options.
Create a general step, ending with a colon, and present the options as bullets.

File: modules/ROOT/pages/installation-prerequisites.adoc [hunk 9]
```diff
@@ -6,34 +6,71 @@ endif::[]
...
-.. Verify that under the *Entitlements* property, the `apiCommunityManager` value is `true`.
-. Identity provider data.
+. Verify that under the *Entitlements* property, the `apiCommunityManager` value is `true`.
+
+== A Domain with Enabled Communities
```
Comment (kbostrom) [inline modules/ROOT/pages/installation-prerequisites.adoc]: Reword to make this a task topic.

File: modules/ROOT/pages/installation-prerequisites.adoc [hunk 10]
```diff
@@ -6,34 +6,71 @@ endif::[]
...
+
+To set up and register a domain:
+
+. Log in to your API Community Manager https://login.salesforce.com/[Salesforce Organization, role=external, window=blank].
```
Comment (kbostrom) [inline modules/ROOT/pages/installation-prerequisites.adoc]:
```suggestion
. Log in to your API Community Manager https://login.salesforce.com/[Salesforce organization, role=external, window=blank].
```

File: modules/ROOT/pages/installation-prerequisites.adoc [hunk 11]
```diff
@@ -6,34 +6,71 @@ endif::[]
...
+. Go to *Salesforce Set-up* by selecting the gear icon and selecting *Set-up*.
+. Search for *My Domain* using the *Quick Find* box and then select *My Domain*.
```
Comment (kbostrom) [inline modules/ROOT/pages/installation-prerequisites.adoc]: Reword per Line 16 above.

File: modules/ROOT/pages/installation-prerequisites.adoc [hunk 12]
```diff
@@ -6,34 +6,71 @@ endif::[]
...
+. Enter a domain name for your organization.
++
+The name can include up to 34 letters, numbers, and hyphens.
+. Select *Check Availability*.
++
+If the name is already taken, choose a different one.
+. Select *Register Domain*.
+
+You will receive a notification email when your domain is registered and your
```
Comment (kbostrom) [inline modules/ROOT/pages/installation-prerequisites.adoc]: present tense: GLOBAL
```suggestion
You receive a notification email when your domain is registered and your
```

File: modules/ROOT/pages/installation-prerequisites.adoc [hunk 13]
```diff
@@ -6,34 +6,71 @@ endif::[]
...
+
+NOTE: After you complete API Community Manager installation and configuration, you can configure a full custom domain
+name, also known as a vanity domain. When you're ready, refer to the Salesforce knowledge article
+https://help.salesforce.com/articleView?id=000336819&type=1&mode=1[Run your Salesforce Community under a custom domain, role=external, window=blank].
+
+Using the domain name you selected in the previous step, follow the steps in
+https://help.salesforce.com/articleView?id=networks_enable.htm&type=5[Enable Salesforce Communities, role=external, window=blank]
+to enable communities inside your Salesforce organization.
+
+Wait to receive the notification email that your domain is registered, and then deploy your domain:
```
Comment (kbostrom) [inline modules/ROOT/pages/installation-prerequisites.adoc]: This is confusing.
We say that you can continue with the next steps...but there are no steps until Line 69, and those are for deploying the domain (where is the registration, subdomain name, and testing?)
After telling the audience to wait for the registration email, we diverge into a note that is about what is essentially a separate topic about vanity naming, and then a para about enabling communities.

Comment (kbostrom) [reply modules/ROOT/pages/installation-prerequisites.adoc]: What we have here is ...:

. Select *Register Domain*.
Wait a few minutes
Continue with these steps while you wait:  (no steps)
NOTE: vanity name
LINK: enable communities
Wait for registration (repeat) 
Then deploy:  (separate task?)
. Log in
. Go to
.
.
.

File: modules/ROOT/pages/installation-prerequisites.adoc [hunk 14]
```diff
@@ -6,34 +6,71 @@ endif::[]
...
+
+. Log in to your API Community Manager https://login.salesforce.com/[Salesforce Organization, role=external, window=blank].
+. Go to *Salesforce Set-up* by clicking the gear icon and selecting *Set-up*.
+. Search for *My Domain* using the *Quick Find* box and then select *My Domain*.
```
Comment (kbostrom) [inline modules/ROOT/pages/installation-prerequisites.adoc]: Repeats steps above...so is a separate task???
See previous edits to these steps in Lines 44-46.

File: modules/ROOT/pages/installation-prerequisites.adoc [hunk 15]
```diff
@@ -6,34 +6,71 @@ endif::[]
...
+. If the screen indicates your domain is ready, select *Log in*.
```
Comment (kbostrom) [inline modules/ROOT/pages/installation-prerequisites.adoc]:
```suggestion
. If the screen indicates that your domain is ready, select *Log in*.
```

**Review by kbostrom — DISMISSED** [review]
CE per request

**Review by dukesphere — APPROVED** [review]

**Review by kbostrom — APPROVED** [review]
Reapproval

**Pull request merged**